src/Templates/NewModule/build.psake.ps1

```diff
@@ -375,6 +410,7 @@ function PromptUserForCredentialAndStorePassword {
 }
 
 function AddSetting {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSShouldProcess', '', Scope='Function')]
```

Comment:
I did try that, putting it outside the function and at the top of the file, but it doesn't seem to like it at all. (Error is: `Unexpected attribute 'System.Diagnostics.CodeAnalysis.SuppressMessage'.`).